salt/modules/kmod.py

```diff
@@ -123,7 +123,16 @@ def available():
         salt '*' kmod.available
     '''
     ret = []
+
     mod_dir = os.path.join('/lib/modules/', os.uname()[2])
+
+    built_in_file = os.path.join(mod_dir, 'modules.builtin')
+    if os.path.exists(built_in_file):
+        with open(built_in_file, 'r') as f:
```

Comment:
Please replace this with `salt.utils.fopen`

Reply:
Done.